lxd/instance/drivers/driver_qemu.go

```diff
@@ -1017,6 +1017,11 @@ func (d *qemu) Start(stateful bool) error {
 		return err
 	}
 
+	// Ensure secureboot is turned off for images that are not secureboot enabled
+	if d.localConfig["image.requirements.secureboot"] == "false" && (d.expandedConfig["security.secureboot"] == "" || shared.IsTrue(d.expandedConfig["security.secureboot"])) {
```

Comment:
Please can you use !shared.IsTrue

Reply:
Hmm, no, that won't work.
If you don't have `security.secureboot` set, the default is `true` but if you do `!shared.IsTrue` that empty string will be causing `IsTrue` to return `false` skipping the test.

Reply:
What I mean is we shouldn't be comparing to "false" literal unless we also compare to "true" literal for consistency.

Reply:
Ah, I was looking at the other half of the check :)

Reply:
Added `shared.IsFalse` which we can then use for this test

Reply:
Thanks